sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -0,0 +1,230 @@
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesWithTopParameter()
+        {
+            TextAnalyticsClient client = GetClient();
+
+            HealthcareOptions options = new HealthcareOptions()
+            {
+                Top = 1,
+                Skip = 0,
+                IncludeStatistics = true
+            };
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(batchDocuments, options);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.AreEqual(1, resultCollection.Count);
+            Assert.AreEqual(3, resultCollection[0].Entities.Count);
+            Assert.IsNotNull(resultCollection[0].Id);
+            Assert.AreEqual("100mg", resultCollection[0].Entities.FirstOrDefault().Text);
+            Assert.AreEqual("Dosage", resultCollection[0].Entities.FirstOrDefault().Category);
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesWithSkipParameter()
+        {
+            TextAnalyticsClient client = GetClient();
+
+            HealthcareOptions options = new HealthcareOptions()
+            {
+                Skip = 1
+            };
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(batchDocuments, options);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.AreEqual(1, resultCollection.Count);
+            Assert.AreEqual(6, resultCollection[0].Entities.Count);
+            Assert.IsNotNull(resultCollection[0].Id);
+            Assert.AreEqual("rapid", resultCollection[0].Entities.FirstOrDefault().Text);
+            Assert.AreEqual("SymptomOrSign", resultCollection[0].Entities.FirstOrDefault().Category);
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesBatchWithErrorTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = new List<string>
+            {
+                "Subject is taking 100mg of ibuprofen twice daily",
+                "Can cause rapid or irregular heartbeat, delirium, panic, psychosis, and heart failure.",
+                "",
+            };
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(documents);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.IsNotNull(resultCollection[0].Id);
+            Assert.IsNotNull(resultCollection[0].TextAnalyticsError);
+            Assert.AreEqual("Document text is empty.", resultCollection[0].TextAnalyticsError.Message);
+            Assert.AreEqual(TextAnalyticsErrorCode.InvalidDocument, resultCollection[0].TextAnalyticsError.ErrorCode.ToString());
```

Comment:
I don't think u need the `ToString` for `ErrorCode`

Reply:
We need this here as `TextAnalyticsErrorCode.InvalidDocument` is a string and it is not able to compare them and results in error

Reply:
Hmm I am confused... ErrorCode is a String... https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsErrorCode.cs

Also, this is how we are doing it in other tests https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeSentimentTests.cs#L399

Not a big deal though... I am just confused